client/src/components/Verejne/index.js

```diff
@@ -42,35 +75,28 @@ const Verejne = ({
         itemClass="page-item"
         linkClass="page-link"
         hideNavigation
-        pageRangeDisplayed={pageCount}
+        pageRangeDisplayed={VEREJNE_PAGE_RANGE}
         activePage={currentPage}
         itemsCountPerPage={VEREJNE_MAX_PAGE_ITEMS}
         totalItemsCount={entitiesLength}
         onChange={setCurrentPage}
       />
     </div>
-    <GoogleMap />
+    <GoogleMap refetch={() => refetch(getEntitiesUrlFromMapReference(mapReference))} />
```

Comment:
just note that again, you'd force a re-render every time Verejne rerenders because you're using and arrow function, but more on passign refetch elsewhere